### DISCUSSION TITLE
Zigbee devices not responding anymore, only after restart

### DISCUSSION THREAD

**Discussion by pbeumer2001** [Koenkk/zigbee2mqtt#15894] [Devices]
Hi,

I`m playing for a week now with HA & Zigbee2MQTT. Testing alot of stuff and options. Last evening I noticed Watchdog was restarting Zigbee2MQTT so I decided to disable watchdog so I could catch the logfile to see what could be the issue. Looking at the logging a can only see that the devices are onresponsive and that no commincation is there anymore. The green light on the dongle will flash when trying to send something but the blue led is off...

I hoped to see something that for example the usb dongle lost its connection or similar....

I`ve took the logfile from the moment it went wrong... Does anybody know what the problem might be?

`error 2023-01-02 11:27:53: Publish 'set' 'system_mode' to 'Radiator Bedroom Lena' failed: 'Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:27:53: Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:27:59: Publish 'set' 'system_mode' to 'Radiator Kitchen' failed: 'Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:27:59: Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:05: Publish 'set' 'system_mode' to 'Radiator Livingroom' failed: 'Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:05: Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:11: Publish 'set' 'system_mode' to 'Radiator Master Bedroom' failed: 'Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:11: Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:17: Publish 'set' 'system_mode' to 'Radiator Office' failed: 'Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:17: Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:23: Publish 'set' 'current_heating_setpoint' to 'Radiator Bedroom Lena' failed: 'Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:23: Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:29: Publish 'set' 'current_heating_setpoint' to 'Radiator Kitchen' failed: 'Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:29: Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:35: Publish 'set' 'current_heating_setpoint' to 'Radiator Livingroom' failed: 'Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:35: Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:41: Publish 'set' 'current_heating_setpoint' to 'Radiator Master Bedroom' failed: 'Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,120]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:41: Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,120]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:28:47: Publish 'set' 'current_heating_setpoint' to 'Radiator Office' failed: 'Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:28:47: Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
debug 2023-01-02 11:29:32: Saving state to file /config/zigbee2mqtt/state.json
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Bedroom Lena/set/system_mode' with data 'off'
debug 2023-01-02 11:30:48: Publishing 'set' 'system_mode' to 'Radiator Bedroom Lena'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Kitchen/set/system_mode' with data 'off'
debug 2023-01-02 11:30:48: Publishing 'set' 'system_mode' to 'Radiator Kitchen'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Livingroom/set/system_mode' with data 'off'
debug 2023-01-02 11:30:48: Publishing 'set' 'system_mode' to 'Radiator Livingroom'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Master Bedroom/set/system_mode' with data 'off'
debug 2023-01-02 11:30:48: Publishing 'set' 'system_mode' to 'Radiator Master Bedroom'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Office/set/system_mode' with data 'off'
debug 2023-01-02 11:30:48: Publishing 'set' 'system_mode' to 'Radiator Office'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Bedroom Lena/set/current_heating_setpoint' with data '5.0'
debug 2023-01-02 11:30:48: Publishing 'set' 'current_heating_setpoint' to 'Radiator Bedroom Lena'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Kitchen/set/current_heating_setpoint' with data '20.0'
debug 2023-01-02 11:30:48: Publishing 'set' 'current_heating_setpoint' to 'Radiator Kitchen'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Livingroom/set/current_heating_setpoint' with data '20.0'
debug 2023-01-02 11:30:48: Publishing 'set' 'current_heating_setpoint' to 'Radiator Livingroom'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Master Bedroom/set/current_heating_setpoint' with data '12.0'
debug 2023-01-02 11:30:48: Publishing 'set' 'current_heating_setpoint' to 'Radiator Master Bedroom'
debug 2023-01-02 11:30:48: Received MQTT message on 'zigbee2mqtt/Radiator Office/set/current_heating_setpoint' with data '5.0'
debug 2023-01-02 11:30:48: Publishing 'set' 'current_heating_setpoint' to 'Radiator Office'
error 2023-01-02 11:30:54: Publish 'set' 'system_mode' to 'Radiator Bedroom Lena' failed: 'Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:30:54: Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:00: Publish 'set' 'system_mode' to 'Radiator Kitchen' failed: 'Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:00: Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:06: Publish 'set' 'system_mode' to 'Radiator Livingroom' failed: 'Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:06: Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:12: Publish 'set' 'system_mode' to 'Radiator Master Bedroom' failed: 'Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:12: Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:18: Publish 'set' 'system_mode' to 'Radiator Office' failed: 'Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:18: Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":2,"datatype":4,"data":[2]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:24: Publish 'set' 'current_heating_setpoint' to 'Radiator Bedroom Lena' failed: 'Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:24: Error: Command 0xa4c1381888a1d50e/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:30: Publish 'set' 'current_heating_setpoint' to 'Radiator Kitchen' failed: 'Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:30: Error: Command 0xa4c1388fce862766/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:36: Publish 'set' 'current_heating_setpoint' to 'Radiator Livingroom' failed: 'Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:36: Error: Command 0xa4c13881a43f480f/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,200]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:42: Publish 'set' 'current_heating_setpoint' to 'Radiator Master Bedroom' failed: 'Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,120]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:42: Error: Command 0xa4c138cf79d74407/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,120]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error 2023-01-02 11:31:48: Publish 'set' 'current_heating_setpoint' to 'Radiator Office' failed: 'Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)'
debug 2023-01-02 11:31:48: Error: Command 0xa4c1385ca3d6b381/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":4,"datatype":2,"data":[0,0,0,50]}]}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (SRSP - AF - dataRequest after 6000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)`

**Comment by rbjoensen** [comment]
Same here, starte 1 week ago

**Comment by lux73** [comment]
Error `SRSP - AF - dataRequest after 6000ms` could be related to this:

https://www.zigbee2mqtt.io/guide/faq/#zigbee2mqtt-crashes-after-some-time

it's worth a try

**Comment by lammetak** [comment]
Same issue here

Using Ubuntu -> Docker
Stick -> Conbee2

Figured out the following

- Removing the device and joining fixes the issue for the device.
- Restarting the container does NOT break the devices not responding.
- Restart the host (VM on ESXI) DOES break the devices.

Tested on version:
- 1.35
- 1.34
- 1.33

**Reply by lammetak** [reply]
Bought a new stick (Sonoff). included all devices and issue is resolved. I guess it is something with z2m in combination with the Conbee2
